### DISCUSSION TITLE
Access with different users

### DISCUSSION THREAD

**Discussion by danielgap** [omkarcloud/google-maps-scraper#214] [General]
I would like to be able to access the frontend with different users and that each user only has access to his output. Any hints on how to achieve this?

**Comment by Chetan11-dev** [comment]
Our Desktop App is ready, you can give it to them.

**Comment by Chetan11-dev** [comment]
or, you can create your own backend and handle it from there.

**Reply by danielgap** [reply]
And I can run only the backend part. Currently I start with python run.py and it starts both the backend and the frontend.

**Reply by Chetan11-dev** [reply]
yes, 
python run.py backend
